### PR TITLE
Removing fifo pipes control code from log

### PR DESCRIPTION
**Pull request by ffinfo**

### CODE REVIEW

**Comment by ffinfo** [comment]
This was to soon, to much is failing on this :(

**Comment by ffinfo** [comment]
Should work now

File: biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/BiopetFifoPipe.scala [hunk 1]
```diff
@@ -110,7 +114,26 @@ class BiopetFifoPipe(val parent: Configurable,
 }
 
 object BiopetFifoPipe {
-  val waitScript =
+
+  def changeScript(file: File, fifos: List[File]): Unit = {
```
Comment (hailiangmei) [inline biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/BiopetFifoPipe.scala]: Maybe add a short doc here explaining the meaning of changeScript?

**Review by akaljuvee — APPROVED** [review]

**Comment by hailiangmei** [comment]
everything looks fine to me. A bit more in code documents could be better.

**Review by hailiangmei — APPROVED** [review]

File: biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/BiopetFifoPipe.scala (hunk 1 above)
Comment (ffinfo) [reply biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/BiopetFifoPipe.scala]: added that now

**Pull request merged**